macros/schema_tests/expression_is_true.sql

```diff
@@ -1,12 +1,18 @@
 {% macro test_expression_is_true(model) %}
 
 {% set expression = kwargs.get('expression', kwargs.get('arg')) %}
+{% set condition = kwargs.get('condition', 'true') %}
```

Comment:
I think `condition` can be supplied as a proper keyword argument here, with the default set to `true`. We use this `kwargs.get` pattern to support both the model-level and column-level test specification syntax, but in the general case, I think it makes to keep arguments in the method signature where possible.

Reply:
I was actually a bit confused by the use of `kwargs` here, but I thought there should be a reason for it and kept it the same. I agree a normal argument is better. I'll send the changes with your suggestions. Thanks!